src/OpenTelemetry.Api/Trace/SpanHelper.cs

```diff
@@ -125,5 +129,43 @@ public static Status ResolveSpanStatusForGrpcStatusCode(int statusCode)
 
             return newStatus;
         }
+
+        /// <summary>
+        /// Helper method that returns Status from <see cref="StatusCanonicalCode"/> to save on allocations.
+        /// </summary>
+        /// <param name="statusCanonicalCode"><see cref="StatusCanonicalCode"/>.</param>
+        /// <returns>Resolved span <see cref="Status"/> for the Canonical status code.</returns>
+        public static Status ResolveCanonicalCodeToStatus(string statusCanonicalCode)
+        {
+            bool success = Enum.TryParse(statusCanonicalCode, out StatusCanonicalCode canonicalCode);
+
+            if (!success)
+            {
+                return DefaultStatus;
```

Comment:
we can simply do return default?

Reply:
`default` will create a new instance every time.